src/main/java/com/onlinebookstore/model/User.java

```diff
@@ -27,4 +36,40 @@ public class User {
     private String lastName;
     @Column(name = "shipping_address")
```

Comment:
Hibernate automatically create the name in snake_case style, so don't need to specify "shipping_address" or "first_name" or "last_name"